### ISSUE TITLE
Text on pages can be resized to 200% while still maintaining form.

### ISSUE THREAD

**Issue by CristinaSolana** [envistaInteractive/itagroup-ecommerce-template#223]
Inconsistencies at 200% enlargement:  Search / Point Range - not easily closable, no close option.  Unable to access My Profile at this size on subpages. On index the arrow does not show after enlargement.  Keyboard focus skips navigation icon and cart at this enlarged size.
